packages/web/pages/api/typedefs/article.ts

```diff
@@ -0,0 +1,174 @@
+import { ApolloError } from "apollo-server-micro";
+import { authenticated } from "./scalars";
+
+export const typeDef = `
+  extend type Query {
+    article(id: ID!): Article
+    articles: [Article]
+  }
+
+  extend type Mutation {
+    createdArticle(articleInput: ArticleCreateInput!): Article
```

Comment:
Follow this convention for all fields
`createArticle (input: CreateArticleInput!): Article!`
Easier to remember whats the input name for a given Mutation.

Reply:
I'd take this one more step and just name them CreateInput since we're already in the article file, what do you say?